source/css/_common/components/sidebar/sidebar-button.styl

```diff
@@ -5,7 +5,7 @@
     display: inline-block;
     padding: 0 15px;
     color: rgb(252, 100, 35);
-    border: 1px solid rgb(252, 100, 35);
+    border: 1px solid rgb(252, 100, 35) !important;
```

Comment:
Relevant to #734. Please, test carefully before make changes like this.

Reply:
A mistake... I'll move this to styles of Mist&Muse...